### PR TITLE
fuzzel: 1.7.0 -> 1.8.1 and librsvg -> nanosvg

### PR DESCRIPTION
**Pull request by rodrgz**
###### Description of changes
Bump to 1.8.1 and makes nanosvg the default svg support, since it is built-in/bundled with fuzzel.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by erikrodrgz** [comment]
With this PR, https://github.com/NixOS/nixpkgs/pull/197648 becomes obsolete due to the update version.

**Comment by erikrodrgz** [comment]
cc @fionera @polykernel

**Comment by thiagokokada** [comment]
> With this PR, #197648 becomes obsolete due to the update version.

But #197648 updates to version 1.8.2, while your PR updates to version 1.8.1.

File: pkgs/applications/misc/fuzzel/default.nix [hunk 1]
```diff
@@ -51,22 +50,21 @@ stdenv.mkDerivation rec {
...
+    "-Dsvg-backend=${if svgSupport then "nanosvg" else "none"}"
   ];
 
   meta = with lib; {
     description = "Wayland-native application launcher, similar to rofi’s drun mode";
     homepage = "https://codeberg.org/dnkl/fuzzel";
-    license = licenses.mit;
-    maintainers = with maintainers; [ fionera polykernel ];
+    license = with licenses; [ mit zlib ];
+    maintainers = with maintainers; [ fionera polykernel erikrodrgz ];
```
Comment (thiagokokada) [inline pkgs/applications/misc/fuzzel/default.nix:67]: It seems that you need to add yourself to `maintainers-list.nix` file.

**Comment by erikrodrgz** [comment]
Closing this PR since https://github.com/NixOS/nixpkgs/pull/197648 and https://github.com/NixOS/nixpkgs/pull/200053 make it obsolete.

**Pull request closed without merging**